inc/sainexthopgroup.h

```diff
@@ -219,6 +300,26 @@ typedef enum _sai_next_hop_group_member_attr_t
      */
     SAI_NEXT_HOP_GROUP_MEMBER_ATTR_MONITORED_OBJECT,
 
+    /**
+     * @brief Member admin state. Allows for disabling a member without removing it.
+     *
+     * Should only be used if the type of owning group is SAI_NEXT_HOP_GROUP_TYPE_L4_SESSION_CONSISTENT
+     *
+     * @type sai_next_hop_group_member_admin_state_t
+     * @flags MANDATORY_ON_CREATE | CREATE_AND_SET
```

Comment:
This is not backward compatible. 
Can this be optional with a default of SAI_NEXT_HOP_GROUP_MEMBER_OPER_STATE_ACTIVE ?

Reply:
Thanks, good point. 
I think you meant ADMIN_STATE (OPER_STATE is READ_ONLY) - and yes, I'll change to optional with default as 'SAI_NEXT_HOP_GROUP_MEMBER_ATTR_ADMIN_STATE_ACTIVE'